### PR TITLE
Fixes - Staff of Change, Dispensing Into Food, Round Report, Telebaton Logging, etc

### PR DESCRIPTION
**Pull request by Krausus**
Fixes:

* Removes two magical projectiles that are old, unused, and redundant.
* Fixes issues with transformations caused by the Staff of Change.
  * Fixes transformed humans dropping all their organs/limbs.
  * Fixes non-humans causing runtimes when transformed into cyborgs.
  * Fixes transfer of non-wizard spells when transforming.
  * Fixes transformation attack log not getting transferred.
  * Fixes transforming into species that aren't intended to be generally playable.
    * This means no more shadowlings or Vox Armalis.
* Fixes being able to load food items into chem dispensers.
* Fixes a couple issues with the round-end report.
  * Fixes wizard spells not being properly listed.
  * Fixes several minor formatting issues with the shadowling round-end report.
* Fixes the traitor panel's "undress" option removing limbs and organs.
* Fixes telescopic batons showing up as "classic baton" in attack logs.

### CODE REVIEW

File: code/datums/mind.dm [hunk 1]
```diff
@@ -1069,8 +1069,14 @@ datum/mind
 		else if (href_list["common"])
 			switch(href_list["common"])
 				if("undress")
-					for(var/obj/item/W in current)
-						current.unEquip(W, 1)
+					if(istype(current, /mob/living/carbon/human))
```
Comment (tigercat2000) [inline code/datums/mind.dm]: should probably be if(ishuman(current))

Comment (Krausus) [inline code/datums/mind.dm]: Huh. I explicitly remembered to use ishuman in one spot, and then forgot to in two.

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 2]
```diff
@@ -220,7 +220,7 @@
 		user << "Something is already loaded into the machine."
 		return
 
-	if(istype(B, /obj/item/weapon/reagent_containers/glass) || istype(B,/obj/item/weapon/reagent_containers/food))
+	if(istype(B, /obj/item/weapon/reagent_containers/glass) || istype(B, /obj/item/weapon/reagent_containers/food/drinks/drinkingglass))
```
Comment (Fox-McCloud) [inline code/modules/reagents/Chemistry-Machinery.dm]: /obj/item/weapon/reagent_containers/food/drinks would be better here.

**Comment by Fox-McCloud** [comment]
Aside from the drinks thing, :+1:

**Comment by TheDZD** [comment]
Some of the wizard issues are due to the fact that we had spell_list inside of mobs, and when Tiger ported /tg/'s new spellbook UI, he also added the var into mind, without removing it from mob. It's probably worth nuking it from mob and transferring it to entirely mob to make maintenance simpler at some point in the future.

File: code/modules/reagents/Chemistry-Machinery.dm (hunk 2 above)
Comment (Krausus) [inline code/modules/reagents/Chemistry-Machinery.dm]: Bah, I saw that the chem master checked for `/drinkingglass` and thought, "Yeah, that sounds right."

**Comment by Krausus** [comment]
@TheDZD -- the addition of `spell_list` to minds (and all of the things it broke) predates the new spellbook UI. In fact, one of the bugs it caused (wizards keeping spells after relearning) was left unfixed specifically because the eventual spellbook port "fixed" it.

I'm not sure if it's at all a good idea to entirely remove spell lists from mobs. It makes sense for wizards, because they can swap their mind around to other bodies, but most "spells" are mob-specific abilities that absolutely shouldn't get transferred around. I literally just fixed a problem caused by transferring them around willy-nilly.

**Pull request merged**